kmippp/kmippp.cpp

```diff
@@ -196,6 +196,14 @@ context::key_t context::op_get(context::id_t id) {
 
 }
 
+bool context::op_activate(context::id_t id) {
+    int result = kmip_bio_activate_symmetric_key(bio_, const_cast<char*>(id.c_str()), id.length());
+    if(result != KMIP_OK) {
+        return false;
+    }
+    return true;
```

Comment:
Can we compress that code to a one-liner?
```cpp
    return kmip_bio_activate_symmetric_key(bio_, const_cast<char*>(id.c_str()), id.length()) == KMIP_OK;
```

Reply:
Well, we can, and C++ allows to return int for bool type... but  it is rather question of style.